pkgs/development/libraries/mathgl/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchurl
+, cmake
+, zlib
+, libpng
+, libGL
+}:
+stdenv.mkDerivation rec {
+  pname = "mathgl";
+  version = "8.0.1";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/mathgl/mathgl-${version}.tar.gz";
+    sha256 = "sha256-yoS/lIDDntMRLpIMFs49jyiYaY9iiW86V3FBKGIqVao=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    zlib
+    libpng
+    libGL
+  ];
+
+  meta = with lib; {
+    description = "A library for scientific data visualization";
+    homepage = "https://https://mathgl.sourceforge.net/";
+    license = with licenses; [ gpl3 lgpl3 ];
+    platforms = platforms.linux ++ platforms.windows;
```

Comment:
Did you try building it for windows?

Reply:
No, but the underlying project mathgl supports Windows.

Reply:
Would you rather this say `platforms = platforms.linux;`? I am not aware of best practices here.

Reply:
Updated to leave out Windows.